bindings/cs/rl.net/FactoryContext.cs

```diff
@@ -20,6 +25,23 @@ public FactoryContext() : base(new New<FactoryContext>(CreateFactoryContext), ne
         {
         }
 
+        public FactoryContext(IEnumerable<byte> vwModelEnumerable) : base(
+            new New<FactoryContext>(() => {
+                var vwModelArray = vwModelEnumerable.ToArray();
+                GCHandle handle = GCHandle.Alloc(vwModelArray, GCHandleType.Pinned);
+                try {
```

Comment:
Is there a way to check if this makes a copy?  Unclear to me...

Reply:
There is a copy happening here but I believe it is necessary since they pass in a List which doesn't have a way to obtain a pointer to it's internal array

Reply:
There is a way to avoid the copy on .NET 5 or above with List<> types, but not generally, no.